app/components/Fields/Asset/index.js

```diff
@@ -19,6 +19,7 @@ export default class Asset extends Component {
     instructions: null,
     defaultValue: null,
     readOnly: false,
+    onChange: ()=>{}
```

Comment:
I'd love it if these default props were consistent; either using `f => f` or `() => {}`. Either one is fine by me, they should just be the same.

Reply:
Done and done!